www/data_module/src/data.config.coffee

```diff
@@ -1,5 +1,10 @@
 class LoggingInterceptor extends Config
```

Comment:
probably need to change the name of class, as it does not only do the logging interception now.